exercises/rational-numbers/canonical-data.json

```diff
@@ -400,6 +400,38 @@
             "r": [0, 1]
           },
           "expected": 1.0
+        },
+        {
+          "uuid": "da137f1c-fc94-4035-9813-94f5152a1102",
+          "description": "Raise a negative real number to a rational number that has been reduced",
+          "property": "expreal",
+          "input": {
+            "x": -1,
+            "r": [1, 1]
```

Comment:
Hey! I tried using that example in my Gleam implementation but stumbled on [this](https://github.com/gleam-lang/stdlib/blob/v0.27.0/src/gleam/float.gleam#L346). This behaviour is the same in Elixir/Erlang. Any thoughts?

Reply:
Yeah... I'm up for dropping all of this tbh. It's starting to feel like we are overcomplicating the exercise and not adding much value. Plus, the initial motivation was to add an error case to our Gleam implementation because the function returns a result. Maybe we could just make that same check that `float.power/2` does and have that be our error case?